File: csharp/Database/ApplicationDatabase/Models/AccountProperties.cs
namespace Database.ApplicationDatabase.Models;

public struct AccountProperties
{
    public Guid AccountId;
    public Guid VerificationToken;
    public DateTime TokenExpiresAt;
    public bool EmailVerified;
    public bool SendEmailNotifications;
    public DateTime CreatedAt;
    public DateTime UpdatedAt;
}

File: legacy/backend/lib/auth_server/dal/models/AccountProperties.go
package models

import (
	"time"

	"github.com/google/uuid"
)

type AccountProperties struct {
	AccountId              uuid.UUID
	VerificationToken      uuid.UUID
	TokenExpiresAt         time.Time
	EmailVerified          bool
	SendEmailNotifications bool
	CreatedAt              time.Time
	UpdatedAt              time.Time
}
